error_handling_on_chain/README.md

```diff
@@ -0,0 +1,127 @@
+    ✔ Verify TA and parse Result w/success (616ms)
+        Success: false
+        Error: expected error
+        Value: null
+    ✔ Verify TA and parse Result w/error (189ms)
+```
+
+## Walkthrough
+
+We cover the basics of how to verify an attested function call in a smart
+contract in the
+[Hello World - Bringing A Blocky AS Function Call Attestation On Chain](../../hello_world_on_chain)
+example.
+
+In this example, we go a step further and show you how to extract the `Result`
+struct from the `successFunc` and `errorFunc` function call attestations in the
+[Error Handling](../error_handling) example.
+
+In [`contracts/User.sol`](contracts/User.sol), we define a `parseResult` function:
+
+```solidity
+function parseResult(
+    string memory resultString
+) public
+{
+    JsmnSolLib.Token[] memory tokens;
+    uint number;
+    uint success;
+    (success, tokens, number) = JsmnSolLib.parse(resultString, 50);
+
+    bool resultSuccess;
+    string memory resultError;
+    string memory valueString;
+
+    for (uint i = 0; i < number; i++) {
+        if (tokens[i].jsmnType == JsmnSolLib.JsmnType.STRING) {
+            string memory key = JsmnSolLib.getBytes(
+                resultString,
+                tokens[i].start,
+                tokens[i].end
+            );
+
+            if (keccak256(bytes(key)) == keccak256("Success")) {
+                resultSuccess = JsmnSolLib.parseBool(
+                    JsmnSolLib.getBytes(
+                        resultString,
+                        tokens[i + 1].start,
+                        tokens[i + 1].end
+                    )
+                );
+            } else if (keccak256(bytes(key)) == keccak256("Error")) {
+                resultError = JsmnSolLib.getBytes(
+                    resultString,
+                    tokens[i + 1].start,
+                    tokens[i + 1].end
+                );
+            } else if (keccak256(bytes(key)) == keccak256("Value")) {
+                valueString = JsmnSolLib.getBytes(
+                    resultString,
+                    tokens[i + 1].start,
+                    tokens[i + 1].end
+                );
+            }
+        }
+    }
+
+    console.log("\tSuccess: %s", resultSuccess);
+    console.log("\tError: %s", resultError);
+    console.log("\tValue: %s", valueString);
+
+    require(resultSuccess, resultError);
+    emit ResultValue(valueString);
+}
+```
+
+which takes in the verified `TAParserLib.FnCallClaims` `claims.Output` and uses
+`JsmnSolLib` to parse the JSON serialized `Result` struct. JSON serialization
+does not guarantee struct field order, so we loop over the tokens to extract the
+fields: `Result.Success`, `Result.Error`, and `Result.Value`. If `Result.Success`
+is `false`, we revert the transaction with `Result.Error` as the error message.
+If `Result.Success` is `true`, we emit `Result.Value` as an event, where
+`Result.Value` itself may be a JSON serialized struct representing the output
+of the function call.
+
```

Comment:
Could you add a step for running the local tests? In hello_world_on_chain we talk about user.ts. In the context of this example, I think it would be helpful to tell the users how we are loading in the attestations from the `inputs` folder

Reply:
added step for running locally along with basic descriptions for what the tests are doing